plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -496,6 +497,14 @@ class FeatureDevController(
                         )
                     ),
                 )
+            } else if (err is ZipFileError) {
```

Comment:
I was expecting to see another condition for `ContentLengthError`, why that's not the case?

Reply:
That is being handled as `RepoSizeError` [here](https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt#L438-L454)